stock_location_content_template/report/stock_location_content_report.py

```diff
@@ -0,0 +1,120 @@
+# Copyright (C) 2022 Open Source Integrators (https://www.opensourceintegrators.com)
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+from odoo import _, fields, models
+from odoo.exceptions import UserError
+
+
+class LocationContentReport(models.TransientModel):
+    _name = "stock.location.content.report"
```

Comment:
```suggestion
    _name = "stock.location.content.template.report"
```
To avoid possible naming collision